content/source/docs/enterprise/private/automating-the-installer.html.md

```diff
@@ -301,10 +301,19 @@ If the installation script (`install.sh`) exits successfully, but the Replicated
 * Verify the locations and permissions of the files. The Replicated configuration file should be placed in `/etc/replicated.conf`, and the application settings file should be placed in the path specified in the Replicated configuration file. The permissions of both files should be `600` if owned by the `replicated` user, or `644`.
 * Run a JSON validator on the files to check their validity.
 
-After resolving any issues with file validity or permissions, re-run the installer to verify that the configuration is applied.
+After resolving any issues with file validity or permissions, run the following commands to reload your configuration:
+
+``` bash
+sudo systemctl stop replicated replicated-ui replicated-operator 
+sudo rm -rf /var/lib/replicated 
+sudo systemctl start replicated replicated-ui replicated-operator
+```
+
```

Comment:
Conceptual yikes, and can you confirm that we've tested this and verified it works?

I think the literal commands are fine for a first pass, but we've been looking at adding our own guide to restarting Replicated, because these commands don't work on every distro, so this is inclining me further to thinking we should make that a part of the public docs.

Reply:
Yeah, if you're already writing it; I would agree on making it public. Hopefully, we can get a `ptfe restart` command in v5 to remedy this in the future

Reply:
The commands will work on any systemd-compatible distro (rhel/ubuntu/whatever). We should make it explicit though that this is ONLY for new installations.

Running `rm -rf /var/lib/replicated` on an existing installation that we care about is *not* a good idea.